contracts/crosschain/axelar/AxelarGatewaySource.sol

```diff
@@ -2,54 +2,58 @@
 
 pragma solidity ^0.8.27;
 
-import {CAIP2} from "@openzeppelin/contracts@master/utils/CAIP2.sol";
-import {CAIP10} from "@openzeppelin/contracts@master/utils/CAIP10.sol";
-import {Strings} from "@openzeppelin/contracts@master/utils/Strings.sol";
+import {CAIP2} from "@openzeppelin/contracts/utils/CAIP2.sol";
+import {CAIP10} from "@openzeppelin/contracts/utils/CAIP10.sol";
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
 import {AxelarGatewayBase} from "./AxelarGatewayBase.sol";
 import {IERC7786GatewaySource} from "../interfaces/draft-IERC7786.sol";
 
 /**
- * @dev Implementation of an ERC7786 gateway source adapter for the Axelar Network.
+ * @dev Implementation of an ERC-7786 gateway source adapter for the Axelar Network.
  *
- * The contract provides a way to send messages to a remote chain using the Axelar Network
+ * The contract provides a way to send messages to a remote chain via the Axelar Network
  * using the {sendMessage} function.
  */
 abstract contract AxelarGatewaySource is IERC7786GatewaySource, AxelarGatewayBase {
     using Strings for address;
 
+    error SendingValueNotSupported();
```

Comment:
This is consistent with the other "UnsupportedAttribute" we have. Also fits vanilla repository guidelines better.

```suggestion
    error UnsupportedNativeTransfer();
```